### DISCUSSION TITLE
Can a G6 sensor be started with xDrip if you use a t:slim pump?

### DISCUSSION THREAD

**Discussion by Navid200** [Navid200/xDrip#175] [General]
I had been told that it was necessary to start the sensor with the pump.  
But, I would like to make sure that is a correct requirement and is not just due to misinformation.
So, would you please explain here if you believe this is or is not a requirements based on your experience or documentation?

**Comment by jonprogers** [comment]
Bringing forward what I said in discussion #165 to keep information complete...

I'm a t:slim user, since August 2019, and always start and stop my sensors with xDrip, t:slim picks the session up no problem at all and with no user intervention as long as it already has the correct tx id.

Only slight wrinkle is that it takes an extra 5 minute cycle for the pump to pick up the session as with firefly transmitters the comms with the tx is completed first for the medical slot then it does the mobile slot afterwards, but for earlier devices I believe they were both instigated together (long time since I used a pre firefly tx!!).

This worked in this way for both BasalIQ and the newer ControlIQ versions of the t:slim software (current software version:- Moonlight v7.4). 

The only difference MAY be that I am a UK user and there is a slight difference in that I share pump data with my endo through Diasend rather than the t:connect software - not sure if that means there is a slightly different software variant in the UK that may have differing quirks...
